controls/V-219162.rb

```diff
@@ -72,5 +72,32 @@
   tag fix_id: 'F-20886r304815_fix'
   tag cci: ['SV-109655', 'V-100551', 'CCI-001851']
   tag nist: ['AU-4 (1)']
+
+  auditd_cfg = input('auditd_cfg')
```

Comment:
Please just call inputs directly. No need to assign them to local variables.

Reply:
Removed the vars. Using inputs directly